tests/test_cases/issue_134/test_reals.py

```diff
@@ -1,43 +0,0 @@
-import logging
```

Comment:
It would be good if these tests could be moved in a standalone PR, rather than being muddled with the rest of this patch. Would you mind filing a separate PR for that first?

Reply:
Ok, no problem